### ISSUE TITLE
SteamVR blocking alvr add-on

### ISSUE THREAD

**Issue by rubinogiov** [polygraphene/ALVR#617]
So, using it with Oculus Quest, nearly every time I start server (press the trigger in the client to start SteamVR or start it in any other way) SteamVR blocks all it's add-ons (including alvr). Unblocking them and restarting doesn't fix it and couldn't find any other way to fix it.

I say "nearly" because it actually did start correctly the first time I tried, but it hasn't any other time.
Only similar issue I could find is this: https://github.com/polygraphene/ALVR/issues/609
but not having an Oculus Link I cannot really try it.

![image](https://user-images.githubusercontent.com/9801675/77585832-748a2c80-6ee5-11ea-9ecd-9599a4fed119.png)
![image](https://user-images.githubusercontent.com/9801675/77585891-8bc91a00-6ee5-11ea-8214-e0d4a991cdd1.png)

Any pointer would be greatly appreciated.

**Comment by mole-cular** [comment]
I had the same problem and I eventually fixed it by using https://github.com/JackD83/ALVR version of ALVR and beta version of SteamVR. I just have to make sure not to launch steam(vr) before alvr and since then everything is working smoothly.

**Comment by DemonsDeath0976** [comment]
witch one are you using for alvr ver

**Comment by rubinogiov** [comment]
> I had the same problem and I eventually fixed it by using https://github.com/JackD83/ALVR version of ALVR and beta version of SteamVR. I just have to make sure not to launch steam(vr) before alvr and since then everything is working smoothly.

Ok, so: ultimately the cable to use Oculus Link arrived, so I didn't try much anymore. But before that I did try with both versions and it did the same thing.
For the sake of others that might have the same problem, I will say that I did so very hurriedly, and I'm not quite sure i reinstalled the client on the headset as well. So trying that version or other different versions of ALVR might still be a solution worth trying.
